packages/babel-core/src/config/helpers/config-api.js

```diff
@@ -86,7 +86,9 @@ function assertVersion(range: string | number): void {
       `it is likely that something in your build process is loading the ` +
       `wrong version. Inspect the stack trace of this error to look for ` +
       `the first entry that doesn't mention "@babel/core" or "babel-core" ` +
-      `to see what is calling Babel.`,
+      `to see what is calling Babel. It is also possible a globally ` +
+      `installed version of babel-cli is being used if neither that package ` +
+      `nor @babel/cli appear inside the package.json`,
```

Comment:
This is never true in this package. Since we will release this as Babel >7.9.x, it's impossible that the globally installed `babel-cli` is being used because it only loads `babel-core` 6.

You should update the message in `@babel/helper-plugin-utils` instead.